styx-common/src/main/java/com/spotify/styx/util/WorkflowValidator.java

```diff
@@ -133,6 +139,13 @@ public static Builder newBuilder(DockerImageValidator dockerImageValidator) {
       }
     });
 
+    // Check if the service account contains space
+    cfg.serviceAccount().ifPresent(serviceAccount -> {
+      if (!validateServiceAccount(serviceAccount)) {
+        e.add("service account format is not valid");
```

Comment:
Let's make this message more informative, like:
```java
e.add("service account is not a valid email address: " + serviceAccount);
```

Reply:
Addressed